src/org/jitsi/impl/neomedia/VideoMediaStreamImpl.java

```diff
@@ -440,10 +440,15 @@ else if(closestAspect == null)
         = new RTCPReceiverFeedbackTermination(this);
 
     /**
+     *
+     */
+    private final PaddingTermination paddingTermination = new PaddingTermination();
+
+    /**
      * The <tt>RemoteBitrateEstimator</tt> which computes bitrate estimates for
      * the incoming RTP streams.
      */
-    private final RemoteBitrateEstimator remoteBitrateEstimator
+    private final RemoteBitrateEstimatorSingleStream remoteBitrateEstimator
```

Comment:
i assume we were returning a broader type before because the caller didn't need to know/care which bitrateestimator was being used...do we need to return the specific type?